pom.xml

```diff
@@ -48,23 +49,29 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <spring.boot.version>1.5.4.RELEASE</spring.boot.version>
         <java.version>1.8</java.version>
+        <bom.version>0.1.4</bom.version>
 
-        <azure.doucmentdb.starter.version>${project.version}</azure.doucmentdb.starter.version>
-        <azure.doucmentdb.autoconfigure.version>${project.version}</azure.doucmentdb.autoconfigure.version>
-        <spring.data.documentdb.version>${project.version}</spring.data.documentdb.version>
-        <azure.storage.starter.version>${project.version}</azure.storage.starter.version>
-        <azure.storage.autoconfigure.version>${project.version}</azure.storage.autoconfigure.version>
-        <azure.servicebus.autoconfigure.version>${project.version}</azure.servicebus.autoconfigure.version>
-        <azure.servicebus.starter.version>${project.version}</azure.servicebus.starter.version>
-        <azure.mediaservices.starter.version>${project.version}</azure.mediaservices.starter.version>
-        <azure.mediaservices.autoconfigure.version>${project.version}</azure.mediaservices.autoconfigure.version>
-        <azure.spring.common.version>${project.version}</azure.spring.common.version>
+        <azure.doucmentdb.starter.version>${bom.version}</azure.doucmentdb.starter.version>
```

Comment:
@rajadilipkolli The project.version property will be evaluated to user's module version, whose version should be independent of our starter version. So, I have to modify your change to the right. The sample projects are examples that user can specify whatever the version they want, not necessarily 0.1.4.